src/olympia/scanners/tasks.py

```diff
@@ -291,7 +291,9 @@ def run_yara_query_rule(query_rule_pk):
         return
     log.info('Fetching versions for run_yara_query_rule on rule %s', rule.pk)
     # Build a huge list of all pks we're going to run the tasks on.
-    qs = Version.unfiltered.filter(addon__type=amo.ADDON_EXTENSION)
+    qs = Version.unfiltered.filter(
+        addon__type=amo.ADDON_EXTENSION, file__file__isnull=False
```

Comment:
is `null` a possible value for the `File.file` field?  (the field definition doesn't include `null=True`)

Reply:
Should it be `file__isnull=False` to filter out missing files instead?

Reply:
It's not a possible value no. I'll fix it.

Reply:
f141214c4108ee0fe0f23cb82d05eaaac0764841